Tests/Unit Tests/JSON Validator/TWTJSONObjectValidatorTestCase.m

```diff
@@ -67,38 +77,55 @@ - (void)testSuite
 - (void)testKnownFailingTests
 {
     // TODO: make these relative
-    NSString *directoryPath = @"/Users/jillcohen/Developer/TWTValidation/Tests/JSONSchemaTestSuite/tests/draft4/";
-    NSError *error = nil;
-
-    for (NSDictionary *test in [self testsInDirectory:directoryPath]) {
-        TWTJSONObjectValidator *validator = [TWTJSONObjectValidator validatorWithJSONSchema:test[TWTTestKeywordSchema] error:nil warnings:nil];
-        for (NSDictionary *testValue in test[TWTTestKeywordTests]) {
-            error = nil;
-            if ([[self failingTests] containsObject:testValue[TWTTestKeywordDescription]]) {
-                BOOL shouldPass = [testValue[TWTTestKeywordValid] boolValue];
-                XCTAssertTrue([validator validateValue:testValue[TWTTestKeywordData] error:&error] == shouldPass, @"\nValue: %@\nSchema: %@\nshould have %@ed because %@. (%@)",
-                              testValue[TWTTestKeywordData], test[TWTTestKeywordSchema], shouldPass ? @"pass" : @"fail", testValue[TWTTestKeywordDescription], test[TWTTestKeywordDescription]);
-            }
-        }
-    }
+//    NSString *directoryPath = @"/Users/jillcohen/Developer/Two-Toasters-GitHub/TWTValidation/Tests/JSONSchemaTestSuite/tests/draft4/";
+//    NSError *error = nil;
+//
+//    for (NSDictionary *test in [self testsInDirectory:directoryPath]) {
+//        TWTJSONObjectValidator *validator = [TWTJSONObjectValidator validatorWithJSONSchema:test[TWTTestKeywordSchema] error:nil warnings:nil];
+//        for (NSDictionary *testValue in test[TWTTestKeywordTests]) {
+//            error = nil;
+//            if ([[self failingTests] containsObject:testValue[TWTTestKeywordDescription]]) {
+//                BOOL shouldPass = [testValue[TWTTestKeywordValid] boolValue];
+//                XCTAssertTrue([validator validateValue:testValue[TWTTestKeywordData] error:&error] == shouldPass, @"\nValue: %@\nSchema: %@\nshould have %@ed because %@. (%@)",
+//                              testValue[TWTTestKeywordData], test[TWTTestKeywordSchema], shouldPass ? @"pass" : @"fail", testValue[TWTTestKeywordDescription], test[TWTTestKeywordDescription]);
+//            }
+//        }
+//    }
```

Comment:
These are tests that are in the public test suite but we don't support (e.g., uniqueness of true/1). It's called "failing tests" and was just a way for me to keep track of it. What's a good way to communicate that going forward?